content/translations.ts

```diff
@@ -8,7 +8,7 @@ const LANGUAGES = new Map(
   })
 );
 
-type Translation = {
+export type Translation = {
```

Comment:
We can replace the type declaration here, as there was a duplicated one in [`libs/types/document.ts`](https://github.com/mdn/yari/blob/006eb85729f34721eea92e5f2146f83dd5c37438/libs/types/document.ts#L111-L115)

Reply:
Good catch, fixed in c00822d4e0.